src/Authentication.Abstractions/AzureEnvironment.cs

```diff
@@ -475,6 +483,16 @@ public AzureEnvironment(IAzureEnvironment other)
         /// </summary>
         public string ResourceManagerUrl { get; set; }
 
+        /// <summary>
+        /// The MSGraph endpoint
+        /// </summary>
+        public string MicrosoftGraphUrl { get; set; }
+
+        /// <summary>
+        /// The token audience need for tokens that target MSGraph endpoints
+        /// </summary>
+        public string MicrosoftGraphEndpointResourceId { get; set; }
+
```

Comment:
I suppose this is no longer used. We should consider removing it.

Reply:
Resolved